site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -494,6 +494,27 @@ public function showStatus($gradeable_id) {
             $peer = true;
         }
 
+        //get all graded gradeables for queue stats
+        $submissions_in_queue = 0;
+        $order = new GradingOrder($this->core, $gradeable, $this->core->getUser(), true);
+        $order->sort("id", "ASC");
+        $graded_gradeables = [];
+        foreach ($order->getSortedGradedGradeables() as $g) {
```

Comment:
The getSortedGradedGradeables already returns an array of GradedGradeables. Why loop through them and add them when you could just set directory or just use this function in line 506?

Reply:
Since you don't need the gradeables sorted you could do a more efficient call. Just directly call getGradedGradeables in DatabaseQueries. The users, team, and sort_keys can all be left as null.